### PR TITLE
Staging and intermediate models for MITxOnline courserun certificates + grades

### PR DESCRIPTION
**Pull request by rachellougee**
## Description
This PR adds staging and intermediate models for the following certificates and grades for MITxOnline
stg__mitxonline__app__postgres__courses_courserungrade
stg__mitxonline__app__postgres__courses_courseruncertificate
stg__mitxonline__app__postgres__courses_programcertificate

int__mitxonline__courserun_certificate
int__mitxonline__program_certificates
int__mitxonline__courserun_grades

## Motivation and Context
https://github.com/mitodl/ol-data-platform/issues/428

## How Has This Been Tested?
Ran `dbt run` and Ran `dbt test`
```
dbt test                                              
17:17:16  Running with dbt=1.2.2
17:17:16  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.open_learning.marts

17:17:16  Found 52 models, 634 tests, 0 snapshots, 0 analyses, 717 macros, 0 operations, 0 seed files, 28 sources, 0 exposures, 0 metrics
17:17:16  
17:17:18  Concurrency: 1 threads (target='qa')

---
17:28:51  
17:28:51  Finished running 634 tests in 0 hours 11 minutes and 34.93 seconds (694.93s).
17:28:51  
17:28:51  Completed successfully
17:28:51  
17:28:51  Done. PASS=634 WARN=0 ERROR=0 SKIP=0 TOTAL=634
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Enhancement (improves on existing behavior)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project. (Did you install and run the pre-commit hooks?)
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/ol_dbt/models/staging/mitxonline/stg__mitxonline__app__postgres__courses_courserungrade.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+, cleaned as (
+    select
+        id as courserungrade_id
+        , course_run_id as courserun_id
+        , user_id
+        , grade as courserun_grade
+        , letter_grade as courserun_letter_grade
+        , set_by_admin as courserungrade_is_set_by_admin
+        , passed as courserungrade_is_passed
```
Comment (abeglova) [inline src/ol_dbt/models/staging/mitxonline/stg__mitxonline__app__postgres__courses_courserungrade.sql:15]: Should this be courserungrade_is_passing? is past sound like the grade is in the past or something

**Review by abeglova — APPROVED** [review]
One minor comment, otherwise lgtm

File: src/ol_dbt/models/staging/mitxonline/stg__mitxonline__app__postgres__courses_courserungrade.sql (hunk 1 above)
Comment (rachellougee) [reply src/ol_dbt/models/staging/mitxonline/stg__mitxonline__app__postgres__courses_courserungrade.sql:15]: Yeah. Although some edx API returns 'passed',  it's better to rename it to 'is_passing'. I will rename it

**Comment by rachellougee** [comment]
Renamed the column and ran the test

**Pull request merged**